packages/launcher/src/renderer/launcher/wallets/WalletIcon.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React, { Component } from 'react'
+import styled from 'styled-components/native'
+import Blockies from 'react-blockies'
+
+type IconSize = 'small' | 'medium' | 'large' | 'x-large' | 'xx-large'
```

Comment:
These should probably be defined in a shared file rather than in a specific icon module.